nginx/cernopendata.conf

```diff
@@ -1,9 +1,40 @@
-server {
 
+# Rewrites uri to lowercase
```

Comment:
This comment is perhaps not necessary, since the same implementation is down below?

(You could have both comments in the PR, and keep only the final code in the config file, to make the final committed code very short...)

Reply:
Removed the alternative implementation from the commit.